pkg/apis/manager/v1beta1/build.sh

```diff
@@ -1,29 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
How we can use this new tool to re-generate files ?

Reply:
we utilize the `buf generate` command to regenerate code changes in `.proto`